frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -64,6 +74,40 @@ function PrecinctTallyReport({
   );
 }
 
+interface BmdPaperBallotsProps {
+  electionDefinition: ElectionDefinition;
+  onAllRendered: (numBallots: number) => void;
+  precinctId: string;
+}
+
+function BmdPaperBallots({
+  electionDefinition,
+  onAllRendered,
+  precinctId,
+}: BmdPaperBallotsProps): JSX.Element {
+  const { election } = electionDefinition;
+  const ballots = generateTestDeckBallots({ election, precinctId });
+
+  useEffect(() => {
+    onAllRendered(ballots.length);
+  }, [ballots, onAllRendered]);
+
+  return (
+    <div className="print-only">
+      {ballots.map((ballot, i) => (
+        <BmdPaperBallot
+          ballotStyleId={ballot.ballotStyleId}
+          electionDefinition={electionDefinition}
+          isLiveMode={false}
+          key={`ballot-${i}`} // eslint-disable-line react/no-array-index-key
```

Comment:
I'd like to avoid using an array index as a key here and in other existing code in this file but want to think through how to best approach this, e.g. hash ballot.votes to create a unique identifier, etc.

https://github.com/votingworks/vxsuite/issues/1864